test/bdd/steps/network.js

```diff
@@ -125,6 +126,7 @@ Given(/^I setup (\d+) node[s]*$/, { timeout: 120000 }, function (nodeCount, done
 
         const newNode = new OtNode({
             nodeConfiguration,
+            appDataBaseDir: this.parameters.appDataBaseDir,
```

Comment:
We need a same in step "I additionally setup (\d+) node[s]*$/", right?

Reply:
True.